microshift_getting-started/microshift-architecture.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="microshift-architecture"]
+= Architecture
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-architecture
+
+toc::[]
+
+== The benefits of running containers on the edge
+
+{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. Because edge and small-form-factor computing are by nature created with only the essentials needed to operate, so is {product-title}.
+
+== Key differences from {OCP}
+* Works offline and in low-connectivity environments
+* You do not need to perform external orchestration
```

Comment:
Can you be more specific about what external orchestration means here? Is it referring to things like the advanced cluster management tool or something else? Like my similar comment, there are cases in OCP where clusters are set up in disconnected environments that don't have any external access for management/orchestration

Reply:
Yes, it isn't managed after install and deployment and there are no consoles, etc. I'll comment these two out for today and work on these bullets in a later PR.